changelogs/unreleased/4348-e2e-scenario-5.yml

```diff
@@ -0,0 +1,9 @@
+change-type: patch
+description: 'Adding automated e2e testing for the Compile Reports'
+issue-nr: 4348
+destination-branches:
+- master
```

Comment:
I don't know what are differences(if any) between scenarios on their own, but doesn't that point to merge that branch to every all of them?

Reply:
I did cherry pick the commit, like the bot was asking me honestly 🙃